core/src/subgraph/registrar.rs

```diff
@@ -620,11 +620,30 @@ async fn create_subgraph_version<C: Blockchain, S: SubgraphStore>(
         "block" => format!("{:?}", base_block.as_ref().map(|(_,ptr)| ptr.number))
     );
 
+    // Entity types that may be touched by offchain data sources need a causality region column.
+    let needs_causality_region = manifest
```

Comment:
I think this could benefit from some test coverage, just making sure that given some manifests, the DeploymentCreate ends up with the right entities